controllers/funding/free-materials/formFields.js

```diff
@@ -5,98 +5,98 @@ module.exports = [
         type: 'text',
         required: true,
         label: {
-            'en': 'Name',
-            'cy': 'Eich enw'
+            en: 'Name',
+            cy: 'Eich enw'
         }
     },
     {
         name: 'yourEmail',
         type: 'email',
         required: true,
         label: {
-            'en': 'Email address',
-            'cy': 'Eich cyfeiriad e-bost'
+            en: 'Email address',
+            cy: 'Eich cyfeiriad e-bost'
         }
     },
     {
         name: 'yourNumber',
         type: 'text',
         required: true,
         label: {
-            'en': 'Phone number',
-            'cy': 'Eich rhif ffôn'
+            en: 'Phone number',
+            cy: 'Eich rhif ffôn'
         }
     },
     {
         name: 'yourAddress1',
         type: 'text',
         required: true,
         label: {
-            'en': 'Address line 1',
-            'cy': 'Eich cyfeiriad llinell 1'
+            en: 'Address line 1',
+            cy: 'Eich cyfeiriad llinell 1'
         }
     },
     {
         name: 'yourAddress2',
         type: 'text',
         required: false,
         label: {
-            'en': 'Address line 2',
-            'cy': 'Eich cyfeiriad llinell 2'
+            en: 'Address line 2',
+            cy: 'Eich cyfeiriad llinell 2'
         }
     },
     {
         name: 'yourTown',
         type: 'text',
         required: true,
         label: {
-            'en': 'Town/city',
-            'cy': 'Eich tref/dinas'
+            en: 'Town/city',
+            cy: 'Eich tref/dinas'
         }
     },
     {
         name: 'yourCounty',
         type: 'text',
         required: false,
         label: {
-            'en': 'County',
-            'cy': 'Eich sir'
+            en: 'County',
+            cy: 'Eich sir'
         }
     },
     {
         name: 'yourPostcode',
         type: 'text',
         required: true,
         label: {
-            'en': 'Postcode',
-            'cy': 'Eich côd post'
+            en: 'Postcode',
+            cy: 'Eich côd post'
         }
     },
     {
         name: 'yourProjectName',
         type: 'text',
         required: true,
         label: {
-            'en': 'Project name',
-            'cy': 'Enw eich prosiect'
+            en: 'Project name',
+            cy: 'Enw eich prosiect'
         }
     },
     {
         name: 'yourProjectID',
         type: 'text',
         required: true,
         label: {
-            'en': 'Project ID number',
-            'cy': 'Rhif adnabod eich prosiect'
+            en: 'Project ID number',
+            cy: 'Rhif adnabod eich prosiect'
         }
     },
     {
         name: 'yourGrantAmount',
-        type: 'text',
+        type: 'number',
```

Comment:
I changed this so that new orders being stored won't need all the number-parsing madness in the Gmail script. In the not-too-distant future we'll change this field to be a dropdown/range selector anyway.